crates/script/src/simulate.rs

```diff
@@ -454,3 +459,54 @@ impl FilledTransactionsState {
         Ok(sequence)
     }
 }
+
+/// Returns the native token symbol for a given chain ID
+fn get_native_token_symbol(chain_id: u64) -> &'static str {
```

Comment:
should use ChainId::native_currency_symbol https://docs.rs/alloy-chains/latest/alloy_chains/enum.NamedChain.html#method.native_currency_symbol